### ISSUE TITLE
[HTML Examples] Text Content ~ dt

### ISSUE THREAD

**Issue by a2sheppy** [mdn/interactive-examples#835]
Need an HTML example for the `dt` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt

## Contributing
https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING-HTML.md

**Comment by wbamberg** [comment]
(I hope you don't mind me jumping in here @a2sheppy )

Thanks for offering to help! Yes, this one was added already, but we haven't been very good at keeping the bug tracker updated. I just went through and closed some bugs, so it ought to be a little better now.

What I'd suggest you do is: take a look at https://docs.google.com/spreadsheets/d/1EC23mRNF7i-Bd29A4wEcFV_iiThL1P-NBtH4lzme8eM/edit#gid=0, and find an element that has "Not started" in the **Status** column, and noone's name in the **Assignee** column.

* If that element already has a bug filed for it, comment in the bug that you'd like to work on it. 
* If that element does not have a bug filed already, file one, and then comment that you'd like to work on it. 

That's probably the best way to reduce the chance of collisions.

If you don't see any elements that look interesting, there's another tab in the spreadsheet for attributes, that we haven't even started yet!

Just ask if you have any questions. And thanks again for your interest in helping us out :-).
